### DISCUSSION TITLE
Migrating WPF application to UNO

### DISCUSSION THREAD

**Discussion by slnsvr** [unoplatform/uno#7789] [Q&A]
Hi,

We have a big WPF application and we want to make it cross platform now so that we can run it on MAC as well. As developing entirely from scratch with a new platform is a big work, we are planning to convert the existing application. So, can we migrate the existing WPF app to UNO to achieve the same? If so, kindly tell us whether there are any porting tools to do that. If no porting tools, kindly suggest how to do this and is that a big task?

**Comment by jeromelaban** [comment]
You might want to take a look at this session from Uno conf from @darenm: https://youtu.be/QBQvE4M04fQ?list=TLGGvPxXPcPnYVIxMjAxMjAyMg&t=5642
